store/Cargo.toml

```diff
@@ -17,19 +17,21 @@ doctest = false
 
 [dependencies]
 anyhow = { version = "1.0" }
-clap = { version = "4.3" , features = ["derive"] }
-deadpool-sqlite = { version = "0.6", features = ["rt_tokio_1"]}
+clap = { version = "4.3", features = ["derive"] }
+deadpool-sqlite = { version = "0.6", features = ["rt_tokio_1"] }
 directories = { version = "5.0" }
 figment = { version = "0.10", features = ["toml", "env"] }
 miden-crypto = { workspace = true }
+miden-lib = { package = "miden-lib", git = "https://github.com/0xPolygonMiden/miden-base.git", branch = "main" }
 miden-node-proto = { path = "../proto" }
 miden-node-utils = { path = "../utils" }
-miden_objects = { workspace = true }
+miden_objects = { workspace = true, features = ["serde"] }
```

Comment:
I was hoping to avoid using `serde` in production, but I guess there is no way around it.